### PR TITLE
created section for third-party archetypes, and added mbmore:RandomEn…

### PR DESCRIPTION
**Pull request by mbmcgarry**
…rich and mbmore:RandomSink to the list

### CODE REVIEW

File: source/user/index.rst [hunk 1]
```diff
@@ -74,6 +75,19 @@ and individuals for use by |Cyclus| users.
...
+
+.. toctree::
+   :maxdepth: 2
+
+   cyclusagents
+
+
+Third-Party Archetypes
+~~~~~~~~~~~~~~~~~~~~~~
 **Bright-lite**
```
Comment (opotowsky) [inline source/user/index.rst]: @mbmcgarry We discussed on the dev call that we won't advertise Bright-lite until it's refactored. Maybe we can just comment this out for now?

Comment (opotowsky) [inline source/user/index.rst]: (Whoops, even though it was already on here)

Comment (mbmcgarry) [inline source/user/index.rst]: If we want to comment out Bright-lite and Cyborg for now then I can do that, but they were already listed on the website prior to this PR, I just reorganized slightly.

**Comment by rwcarlsen** [comment]
pinging the original reviewer - @opotowsky...

**Comment by opotowsky** [comment]
Sorry I forgot to come back to this, @mbmcgarry - in now!

**Pull request merged**